public/data/object/processes.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "categories": [],
+    "categories": ["material"],
```

Comment:
Shouldn’t we add the `material` category to the `Composant en plastique (PP)` too?